packages/dds/tree/api-report/tree.api.md

```diff
@@ -1577,7 +1577,7 @@ export interface SequenceFieldEditBuilder {
     move(sourceIndex: number, count: number, destIndex: number): void;
 }
 
-// @internal
+// @beta
 export class SharedTree implements ITree {
```

Comment:
This is our faux 'SharedTree' class we expose to comply with the AzureClient pattern, which requires a class with a specific ctor signature.